pkg/cri/store/stats/stats.go

```diff
@@ -1,27 +1,71 @@
-/*
-   Copyright The containerd Authors.
+package stats
 
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
+import (
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
let's avoid depending on CRI here, the fields in the struct should be agnostic to CRI and "containerd" "private view" of what container stats look like.